array-manipulation-3/src/difference.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-unused-vars */
+function difference(first, second) {
+  let joinedArray = first.concat(second);
+
+  joinedArray = unique(joinedArray);
+
+  for (let i = 0; i < first.length; i++) {
+    for (let y = 0; y < second.length; y++) {
+      for (let x = 0; x < joinedArray.length; x++) {
+        if (first[i] === joinedArray[x] && second[y] === joinedArray[x]) {
+          joinedArray.splice(x, 1);
+        }
+      }
+    }
+  }
```

Comment:
This is _a lot_ of looping. Try starting with an empty array. Then loop through each of your inputs one-by-one. If an element exists in the first array, but not the second, then add it to the empty array. Then do the same for the second input.